examples/flutter_gallery/lib/demo/playground/home.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'playground_demo.dart';
+
+class PlaygroundScaffold extends StatelessWidget {
+  const PlaygroundScaffold({
+    this.title,
+    this.demos,
+  });
+
+  final String title;
+  final List<PlaygroundDemo> demos;
+
+  static final GlobalKey<ScaffoldState> _scaffoldKey =
```

Comment:
Is this key really needed? If it is, please provide a comment (and remove the line break).